src/entities/change/services/index.js

```diff
@@ -0,0 +1,69 @@
+import { createChangeEntity } from "../changeEntity";
+
+const VITE_GITHUB_TOKEN = import.meta.env.VITE_GITHUB_TOKEN;
+const NO_NEW_LINE_MESSAGE = "\\ No newline at end of file";
+
+const getDiff = async (owner, repo, ref) => {
+  const commitData = {
+    owner: owner,
+    repo: repo,
+    ref: ref,
+  };
+
+  const commitUrl = `https://api.github.com/repos/${commitData.owner}/${commitData.repo}/commits/${commitData.ref}`;
+
+  const response = await fetch(commitUrl, {
+    headers: {
+      Authorization: `token ${VITE_GITHUB_TOKEN}`,
+      Accept: "application/vnd.github.diff",
+    },
+  });
+
+  const changedCode = await response.text();
+
+  return getChanges(changedCode);
+};
+
+const getChanges = (diffPerCommit) => {
+  const codeListPerFile = diffPerCommit.split("diff --git").slice(1);
+  const commitChangesMap = new Map();
+
+  for (const commitCode of codeListPerFile) {
+    const result = makeAChange(commitCode);
+    commitChangesMap.set(result.fileName, result.changes);
+  }
```

Comment:
for문으로 표현하는 것과 map으로 표현하는 것 중 저는 map을 선호하는데, 여러분은 어떤 것이 더 좋으신가요?
```suggestion
  codeListPerFile.map((commitCode) => {
    const result = makeAChange(commitCode);
    commitChangesMap.set(result.fileName, result.changes);
  }
```

Reply:
map을 사용할 때 코드를 읽을 때 바로 의미를 파악할 수 있어서 좋네요! 수정하겠습니다~!